toolchain/sem_ir/type_iterator.h

```diff
@@ -59,14 +59,23 @@ class TypeIterator {
   struct SymbolicType {
     SemIR::TypeId facet_type_id;
   };
-  // A work item to mark a non-type value.
-  struct NonTypeValue {
-    // The type of the value.
-    SemIR::TypeId type_id;
+  // A work item to mark a concrete non-type value.
+  struct ConcreteNonTypeValue {
+    SemIR::InstId inst_id;
+  };
+  // A work item to mark a symbolic non-type value.
+  struct SymbolicNonTypeValue {
+    SemIR::InstId inst_id;
+  };
+  // Work item to mark the name of a struct field.
```

Comment:
```suggestion
  // A work item to mark the name of a struct field.
```

Just suggesting for consistency.

Reply:
Done